alvr/dashboard/src/dashboard/mod.rs

```diff
@@ -296,6 +310,95 @@ impl eframe::App for Dashboard {
                 });
         }
 
+        let shutdown_alvr = || {
+            self.data_sources.request(ServerRequest::ShutdownSteamvr);
+
+            crate::steamvr_launcher::LAUNCHER
+                .lock()
+                .ensure_steamvr_shutdown();
+        };
+
+        if let Some((version, message)) = &self.version_modal_data {
+            let no_remind_button =
+                ModalButton::Custom("Don't remind me again for this version".to_string());
+
+            let result = alvr_gui_common::modal(
+                context,
+                "New ALVR version available",
+                Some(|ui: &mut Ui| {
+                    ui.horizontal(|ui| {
+                        ui.add_space(10.0);
+
+                        ui.vertical(|ui| {
+                            ui.heading(format!("ALVR v{version}"));
+
+                            ui.horizontal(|ui| {
+                                ui.spacing_mut().item_spacing.x = 5.0;
+                                ui.style_mut().spacing.button_padding = egui::vec2(10.0, 4.0);
+
+                                ui.heading("You can download this version using the launcher:");
+
+                                if ui.button("Open Launcher").clicked() {
+                                    let layout = crate::get_filesystem_layout();
+                                    let mut success = false;
+                                    if let Some(path) = layout.launcher_exe() {
+                                        if Command::new(path).spawn().is_ok() {
+                                            shutdown_alvr();
+
+                                            success = true;
+                                        }
+                                    }
+
+                                    if !success {
+                                        ui.output_mut(|out| {
+                                            out.commands.push(OutputCommand::OpenUrl(
+                                                OpenUrl::same_tab(
+                                                    "https://github.com/alvr-org/ALVR/releases",
+                                                ),
+                                            ));
+                                        });
+                                    }
+                                }
```

Comment:
This is a good idea ig, I'd still rather not show the button if alvr wasn't installed through the launcher tho. And if they installed the launcher it might be confusing when it doesn't actually open the launcher even tho they have it installed.